Drop-In/src/main/java/com/braintreepayments/api/DropInClient.java

```diff
@@ -428,4 +441,9 @@ void getVaultedPaymentMethods(final FragmentActivity activity, final GetPaymentM
             });
         });
     }
+
+    void setLastUsedPaymentMethodType(PaymentMethodNonce paymentMethodNonce) {
+        Context context = braintreeClient.getApplicationContext();
```

Comment:
Yeah I agree. I know we talked about `getApplicationContext` on `BraintreeClient` being less than ideal. I'll refactor to pass in a `Context` since it's internal.